diff -up Cbc-2.8.9/src/CbcSolver.cpp.orig Cbc-2.8.9/src/CbcSolver.cpp
--- Cbc-2.8.9/src/CbcSolver.cpp.orig	2014-03-28 13:18:14.586120587 -0300
+++ Cbc-2.8.9/src/CbcSolver.cpp	2014-03-28 13:18:48.302121879 -0300
@@ -1862,8 +1862,8 @@ int CbcMain1 (int argc, const char *argv
 	    sprintf(generalPrint + strlen(generalPrint),
 		    "Build Date: %s \n", __DATE__);
 #ifdef CBC_SVN_REV
-	    sprintf(generalPrint + strlen(generalPrint),
-		    "Revision Number: %d \n", CBC_SVN_REV);
+	    /*sprintf(generalPrint + strlen(generalPrint),
+		    "Revision Number: %d \n", CBC_SVN_REV);*/
 #endif
             generalMessageHandler->message(CLP_GENERAL, generalMessages)
             << generalPrint
